clipboard_windows.go

```diff
@@ -242,7 +242,7 @@ func writeImage(buf []byte) error {
 			// yhat := int(height) - y
 			// r, g, b, a := img.At(xhat, yhat).RGBA()
```

Comment:
How does the `-1` changes the overall situation? Could you maybe explain it a bit more?

Reply:
The function img.At() should scan 2D-array from img[0, 0] to img[width-1, height-1].

But Y-axis range in the code `r, g, b, a := img.At(x, height-y).RGBA()` scans from 1 to height,
because of `for y := 0; y < height; y++` returns from 0 to height-1.
`heigh-0 = height, height-(height-1) = 1`

so offset -1 added to y of img.At()

```
img
                  ( ( (
                  w w w
                  - - -
                  3 2 1
      0 1 2 3 ... ) ) )
    0 o o o o ... o o o    -+
    1 o o o o ... o o o -+  |
    2 o o o o ... o o o  |  |
    3 o o o o ... o o o  |  |
    .                    |  |
    .                    |  |
    .                    |  |
(h-3) o o o o ... o o o  |  |
(h-2) o o o o ... o o o  |  |
(h-1) o o o o ... o o o  | -+ img.At(x, height-1-y)
(  h) x x x x ... x x x -+ img.At(x, height-y)
```

Reply:
Thanks for the review.

CI test results show the same problem will exist in readImage().

https://github.com/golang-design/clipboard/blob/5e37dbe251b4636ad9d08eaf36678826b606e56d/clipboard_windows.go#L139

I'll check it.